docs/release/release-2022-08-10.md

```diff
@@ -0,0 +1,308 @@
+
+For example, the following used to define a request body of type `string`, but now defines a request body that is an object with a property named `body` of type string.
+
+```javascript
+op create(body: string): void;
+
+```
+
+To get the previous behavior, the parameter now needs to be explicitly marked with `@body`:
+
+```javascript
+op create(@body body: string): void;
+
+```
+
+## OkResponse is no longer a template
+
+Previously, OkResponse took an argument for the body type. Now it is a simple model like the other XxxResponse types. Alone, it implies a status code of 200 with no body.
+
+Since 200 is the default status code for non-empty bodies, you can usually replace `OkResponse<T>` with simply `T`. If you wish to be more explicit, you can use `OkResponse & T`.
+
+```javascript
+op get(id: string): OkResponse<Pet>;
+
+```
+
+Can be:
+
+```javascript
+op get(id: string): Pet;
+
+```
+
+Or:
+
+```javascript
+op get(id: string): OkResponse & Pet;
+
+```
+
+In certain situations where the body type is not (necessarily) a model, you will need to use the new `Body<T>` type. For example.
+
+```javascript
+op list(): OkResponse<Pet[]>;
+
+```
+
+Can become:
+
+```javascript
+op list(): OkResponse & Body<Pet[]>;
+
+```
+
+Since 200 status code is used by default, this could also be:
+
+```javascript
+op list(): Pet[];
+
+```
+
+Generic models based on `OkResponse<T>` may also require `Body<T>`. For example:
+
+```javascript
+model MyResponse<T> {
+  ...OkResponse<T>;
+  @header example: string;
+}
+
+```
+
+Since T is not constrainted to be a model, it might be an intrinsic type, an array, or the like, the template should be changed to use `Body<T>`:
+
+```javascript
+model MyResponse<T> {
+  ...OkResponse;
+  ...Body<T>;
+  @header example: string;
+}
+
+```
+
+In general, the prior `OkResponse<T>` is equivalent to `OkResponse & Body<T>` now or, equivalently, `{ ...OkResponse, ...Body<T> }`. In practice there are many situations where you can leave out OkResponse altogether and use plain `T` rather than `Body<T>`.
+
+See also https://github.com/microsoft/cadl/blob/main/docs/tutorial.md#request--response-bodies
+
+## Route resolution changes
+
+Resolving operation routes now follows the following logic:
+
+- if there is a service namespace specified
+  - only emit the operations and interfaces under that namespace(recursively)
+- if not:
+  - only emit the operations and interfaces defined at the root (DO NOT look into namespaces)
+
+### Action if applicable
+
+- If a cadl specused a service namespace without `@serviceTitle` add the `@serviceTitle` decorator to the service namespace, otherwise no routes will be emitted.
```

Comment:
```suggestion
- If a cadl spec used a service namespace without `@serviceTitle` add the `@serviceTitle` decorator to the service namespace, otherwise no routes will be emitted.
```